.github/workflows/build-and-upload-archives-on-schedule.yml

```diff
@@ -0,0 +1,22 @@
+name: Venice Publication Pipeline
+
+on:
+  schedule:
+    - cron: "30 0 * * 3"
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    if: github.repository == 'linkedin/venice'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'microsoft'
+          java-version: '11'
```

Comment:
Some partners are still using JDK8, so I am wondering whether JDK11 build could break the existing users.
Can we specify the release_version as 8?

Reply:
Nisarg has answered this and `sourceCompatibility` will fix this.

Reply:
yes, we dont need to set it up here.